Module/dlg/0.dlg.json

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
This is another shop dialog from the looks of it. Please consolidate it down as well.